book/src/volume-cloning.md

```diff
@@ -0,0 +1,48 @@
+# Volume Cloning
+
+## Status and Releases
+
+** Status:** Alpha
```

Comment:
Can you also put the status in the table, similar to: https://kubernetes-csi.github.io/docs/topology.html

Reply:
Yep, I'll actually match how you have the table in topology

Reply:
And this line can be removed now :-)

Reply:
DeRp!